source/tutorials/kubernetes.rst

```diff
@@ -180,71 +350,143 @@ Configure and run CRI-O + kata-runtime
        handler: kata
        EOF
 
+   If you are using the *flannel* for pod networking (see `Choose a pod
+   network add-on`_), with CRI-O + Kata Containers as the runtime, edit the
+   :file:`/etc/crio/crio.conf` file to include:
 
-Install pod network add-on
-**************************
+   .. code-block:: console
 
-You must choose and install a `pod network add-on`_ to allow your pods to communicate. Check whether or not your add-on requires special flags when you initialize the master control plane.
+      [crio.runtime]
+      manage_network_ns_lifecycle = true
```

Comment:
This has been the default setting when you enable the crio system service.  I would specifically point out that flannel is the only exception if that is true. I am not sure if it is.

Reply:
Yeah, I was waffling on this too and how to fit it in. The exception cases are what makes it hard to word but will give it another shot.